### ISSUE TITLE
Classification Accuracy 0.004945164506480558

### ISSUE THREAD

**Issue by jiangwx** [jakc4103/DFQ#12]
I tried 
```
 python3 main_cls.py --quantize --relu --equalize --correction
```
However, I got only 0.004945164506480558 accuracy
The log is given below:
```
jiangwx@SARS-AORUS:~/DFQ$ python3 main_cls.py --quantize --relu --equalize --correction
load weight: modeling/classification/mobilenetv2_1.0-f2a8633.pth.tar
register <class 'torch.nn.modules.conv.Conv2d'> <class 'utils.quantize.QuantNConv2d'>
register <class 'torch.nn.modules.linear.Linear'> <class 'utils.quantize.QuantNLinear'>
register <class 'torch.nn.modules.activation.ReLU6'> <class 'torch.nn.modules.activation.ReLU'>
Setting num_bits for targ layers...
Start cross layer equalization
Start bias correction
Quantizing Layer parameters
SET QUANT MIN MAX
Start inference
Acc: 0.004945164506480558
```

**Comment by jakc4103** [comment]
Works fine here. Is your validation dataset splited into folders following #3 ?

**Comment by jiangwx** [comment]
Thank you very much!
It works fine after splitting the validation dataset, and I get 0.70876 accuracy.
